images/manageiq-ui-worker/Dockerfile

```diff
@@ -18,6 +18,7 @@ COPY container-assets/manageiq-http.conf /etc/httpd/conf.d
 
 RUN source /etc/default/evm && \
     /usr/bin/generate_rpm_manifest.sh && \
+    dnf install -y mod_ssl && \
```

Comment:
Can we put this in a separate section on its own (or even in the base image)?  Reason I'm asking is because this section is copy-pasted across the images, so for consistent updates I'd like to keep them all the same.

Reply:
Additionally, we should install mod_ssl _before_ we generate the manifest that will need to include it.

Reply:
Moved to https://github.com/ManageIQ/manageiq-rpm_build/pull/163 where we install httpd